easysid.cpp

```diff
@@ -152,42 +157,66 @@ EXPORT void set_filter_res(uint8_t res)
     sid.write(23, reg);
 }
 
-EXPORT void set_filter_enable(uint8_t voice)
+EXPORT void set_filter(uint8_t voice, bool enable)
 {
     if (voice < 1 || voice > 3) {
         return;
     }
     char reg = sid.read_state().sid_register[23];
-    reg &= ~0x0f;
-    reg |= 1 < (voice - 1);
+    reg &= ~1 << (voice - 1);
```

Comment:
This still isn't right. The above expression does not give the intended results. The precedence rules of C++ are here for reference: http://en.cppreference.com/w/cpp/language/operator_precedence